src/global/general.R

```diff
@@ -15,7 +15,7 @@ unprod <- univ_products %>%
 
 # Load spatial files from Drive if not already held on local machine
 # (takes a long time)
-load_spatial_data()
+# load_spatial_data()
```

Comment:
@spencerrhea You mentioned a `//` issue associated with this function's output files, but as it stands, those files aren't being re-downloaded if they already exist, right? When I run this it just says, "All spatial files from GDrive are already held locally." and then returns. Small potatoes, but it's nice if the whole system can build from scratch, with no need to manually toggle code.